app/src/main/java/com/example/mycleaninglog/MainActivity.kt

```diff
@@ -42,6 +38,7 @@ class MainActivity : ComponentActivity() {
 @ExperimentalMaterialApi
 @Composable
 fun HomeScreen() {
+    // calls the 1st level expandable cards. hardcoded because they should never change. These make up the main menu
```

Comment:
Inline comments are frowned upon these days.  Readable code should not require heavy comments.  So... consider converting this to KDoc and putting it above the function.